src/js/angular/import/controllers/tab.controller.js

```diff
@@ -25,38 +25,39 @@ function TabController($scope, $location, ImportViewStorageService, ImportContex
     // Public functions
     // =========================
 
+    const updateHelpVisibility = () => {
+        let empty = checkEmptyState[$scope.activeTabId] && areResourcesEmpty();
+        let helpVisible = ImportViewStorageService.getIsHelpVisible($scope.activeTabId);
```

Comment:
you should have lint warnings here ^. empty and helpVisible should be `const` and not `let`